backend/routes/PostReaction/ShareHandler.py

```
@@ -0,0 +1,93 @@
+    """Get post details from share token"""
+    shared_post = db.query(Share).filter(Share.share_token == share_token).first()
+    if not shared_post:
+        raise HTTPException(status_code=404, detail="Invalid or expired share link")
+
+    post = db.query(Post).filter(Post.id == shared_post.post_id).first()
+    if not post:
+        raise HTTPException(status_code=404, detail="Post not found.")
+    
+    return post
+
+def get_post_additional_data(db: Session, post: Post, current_user_id: int):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_post_additional_data`

(ANN201)